[Images] Several improvements to text (#5397)
cloudflare/cloudflare-docs: content/images/cloudflare-images/images-analytics.md

If you are an Enterprise customer, contact your Customer Success Manager to increase your Cloudflare Images' quota and avoid this error. You can also [delete images](/images/cloudflare-images/transform/delete-images/) from your account.

> Other customers can select the **Change Subscription** button to change their account type, increase their quota limit and avoid this error, or [delete images](/images/cloudflare-images/transform/delete-images/) from their account.

Reviewer: ```suggestion
Non-Enterprise customers can select **Change Subscription** to change their account type, increase their quota limit and avoid this error, or [delete images](/images/cloudflare-images/transform/delete-images/) from their account.
```
